exercises/robot-name/RobotNameTest.cs

```diff
@@ -30,4 +31,20 @@ public void Can_reset_the_name()
         robot.Reset();
         Assert.NotEqual(originalName, robot.Name);
     }
+
+    [Fact(Skip = "Remove to run test")]
+    public void After_reset_the_name_is_valid() {
+        robot.Reset();
+        Assert.Matches(@"^[A-Z]{2}\d{3}$", robot.Name);
+    }
+
+    [Fact(Skip = "Remove to run test")]
+    public void Robot_names_are_unique() {
```

Comment:
The existing [example implementation](https://github.com/exercism/csharp/blob/master/exercises/robot-name/Example.cs) fails this new test: https://travis-ci.org/exercism/csharp/builds/485576414?utm_source=github_status&utm_medium=notification Could you modify the example implementation to have it pass this test?